ansible/environments/local/group_vars/all

```diff
@@ -4,8 +4,10 @@ consul_conf_dir: /tmp/consul
 whisk_logs_dir: /tmp/wsklogs
 cli_conf_dir: /tmp/cli
 go_cli_conf_dir: /tmp/go-cli
+go_cli_nginx_dir: "{{ nginx_conf_dir }}/cli/go/download"
 docker_registry: ""
 docker_dns: ""
+docker_build_time_out: "240"
```

Comment:
I was trying to resolve the timeout issue:

It just takes a long time to build all the cli for different operating systems, like mac, linux, windows, etc.
Our travis CI only takes 10 mins as the timeout, so it always fails within 10 mins without finishing the build. Anyone knows how to extend the timeout?


Here is the log I have got in travis ci:
```
TASK [go-cli : Pull the latest image of go-cli] ********************************

Thursday 23 June 2016  16:19:40 +0000 (0:00:00.385)       0:01:47.867 ********* 

skipping: [172.17.42.1]

TASK [go-cli : Build Docker image] *********************************************

Thursday 23 June 2016  16:19:41 +0000 (0:00:00.133)       0:01:48.001 ********* 

No output has been received in the last 10m0s, this potentially indicates a stalled build or something wrong with the build itself.

The build has been terminated
```